examples/run/run.cpp

```diff
@@ -68,15 +81,24 @@ class Opt {
             return 2;
         }
 
+        ctx_params_.n_batch        = context_size_ >= 0 ? context_size_ : context_size_default;
+        model_params_.n_gpu_layers = ngl_ >= 0 ? ngl_ : ngl_default;
+        temperature_               = temperature_ >= 0 ? temperature_ : temperature_default;
+
         return 0;  // Success
     }
 
+    llama_context_params         ctx_params_;
```

Comment:
```suggestion
    llama_context_params         ctx_params;
```

The trailing underscore does not have clear meaning to it, I'd suggest to remove it or either add postfix, for example: `ctx_params_current`

Reply:
Removed all trailing _ 's